salt/modules/gpg.py

```diff
@@ -783,16 +783,21 @@ def import_key(text=None, filename=None, user=None, gnupghome=None):
 
 
 def export_key(
-    keyids=None, secret=False, user=None, gnupghome=None, use_passphrase=False
+    keyids=None,
+    secret=False,
+    user=None,
+    gnupghome=None,
```

Comment:
can you make sure to keep the kwargs order in case anyone is currently calling this functional with positional args. (`use_passphrase` should go before `output`.

Reply:
How do I determine what the correct order is.
The `python-gnu` export_keys function https://github.com/vsajip/python-gnupg/blob/master/gnupg.py
Is defined as :
    ```def export_keys(self, keyids, secret=False, armor=True, minimal=False, passphrase=None, expect_passphrase=True, output=None):```
Do you want me to keep that order? The salt `export_key` function uses kwargs that aren't found in the `python-gnupg` `export_keys` function. 
I'm just confused as to how you want them sorted. 
Can you please tell me what order you want them to be in? (that might be easier)

Reply:
Nevermind, I think I understand what you're saying.